index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <header>
+    <body>
+      <main>
+        <section>
+          <article>
+            <div>
+              <p>Order Id: 123098097124</p>
+              <p>Placed On: 11/12/2021</p>
+            </div>
+            <div>
+              <p>View Details</p>
+            </div>
+            <div>
+              <p>-----------------------------------------------------------------------------------</p>
+            </div>
+          </article>
+          <article>
+            <div>
+              <h1>Smart Desk</h1>
+              <p>Quantity: 1</p>
+              <p>Design: 53" X 29" Classic</p>
+              <p>Top: White Matte</p>
+              <p>Frame: White</p>
+              <p>Price: 499$</p>
+            </div>
+            <div>
+              <img src="https://github.com/StevieAmb/receipt-comp/blob/main/images/thestandingdesk.jpg?raw=true">
+            </div>
+            <div>
+              <p>-----------------------------------------------------------------------------------</p>
+            </div>
+          </article>
+          <article>
+            <div>
```

Comment:
I wonder if these might have to be buttons, instead of individual divs, and I target the p element with the numbers in the CSS to style them all the same.